### PR TITLE
[YUNIKORN-2026] Update Chinese translation of feature document

### PR DESCRIPTION
**Pull request by targetoee**
### What is this PR for?
In the feature document, some parts are only present in the English document and are missing in the Chinese translation. This pr aims to fill in the omitted sections.

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
[YUNIKORN-2026](https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-2026)

### CODE REVIEW

**Review by steinsgateted — APPROVED** [review]
LGTM

**Comment by wilfred-s** [comment]
This one slipped by without thinking about the translation: https://github.com/apache/yunikorn-site/pull/375/files
Can this be added so we can commit and get the two pages in-sync with one commit?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 1]
```diff
@@ -69,3 +78,14 @@ YuniKorn会自动为未完成的请求进行资源预留。
 吞吐量是衡量调度器性能的关键标准。这对于一个大规模的分布式系统来说是至关重要的。
 如果吞吐量不好，应用程序可能会浪费时间等待调度，并进一步影响服务的 SLA（服务级别协议）。
 集群越大，对吞吐量的要求也越高。[基于Kube标记的运行评估](performance/evaluate_perf_function_with_kubemark.md) 章节显示了一些性能数据。
+
+## 应用程序数量限制
+
+应用程序数量限制允许用户設定已配置队列里运行中的应用程序数量上限。
```
Comment (zhuqi-lucas) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:84]: 設定 change to 设定

**Review by zhuqi-lucas — CHANGES_REQUESTED** [review]

**Comment by wilfred-s** [comment]
No longer relevant after [YUNIKORN-2411](https://issues.apache.org/jira/browse/YUNIKORN-2411)

**Pull request closed without merging**